decoder/src/lib.rs

```diff
@@ -751,23 +751,23 @@ mod tests {
             0b1110_0101, // u8
         ];
         decode_and_expect(
-            "x: {0=0..4:b}, y: {0=3..8:b}",
+            "x: {0=0..4:b}, y: {0=3..8:#b}",
             &bytes,
-            "0.000002 INFO x: 0b101, y: 0b11100",
+            "0.000002 INFO x: 101, y: 0b11100",
         );
     }
 
     #[test]
     fn bitfields_reverse_order() {
         let bytes = [
-            0,           // index
-            2,           // timestamp
-            0b1101_0010, // u8
```

Comment:
I think you should keep the `0b` here.

Reply:
Yes, this fixes the host-tests.

Reply:
oops this looks like a mistake on my part